vivisect/__init__.py

```diff
@@ -49,6 +50,10 @@
 
 import vivisect.analysis.generic.emucode as v_emucode
 
+def guid():
```

Comment:
i guess i did miss uuid.uuid4() being the "random" uuid (which is basically what a guid is).  it's kinda obtuse that they name them that way.